jenkins-client-it-docker/jobs/test/config.xml

```diff
@@ -16,6 +16,15 @@
       <command>echo &quot;test&quot;</command>
     </hudson.tasks.Shell>
   </builders>
-  <publishers/>
+  <publishers>
+    <hudson.tasks.ArtifactArchiver>
+      <artifacts>**/*.txt</artifacts>
+      <allowEmptyArchive>false</allowEmptyArchive>
+      <onlyIfSuccessful>true</onlyIfSuccessful>
+      <fingerprint>false</fingerprint>
+      <defaultExcludes>true</defaultExcludes>
+      <caseSensitive>true</caseSensitive>
+    </hudson.tasks.ArtifactArchiver>
+  </publishers>
```

Comment:
Can you explain why you have change the content of that file?

Reply:
Well, I need a job that actually does archive some files to get a proper Artifact object in NoExecutorStartedGetJobDetailsIT#shouldCheckTheBuildCause().